javascript/ql/src/semmle/javascript/ES2015Modules.qll

```diff
@@ -235,7 +235,7 @@ abstract class ExportDeclaration extends Stmt, @export_declaration {
   ES2015Module getEnclosingModule() { this = result.getAnExport() }
 
   /** Holds if this export declaration exports variable `v` under the name `name`. */
-  abstract predicate exportsAs(LexicalName v, string name);
+  abstract predicate exportsAs(LexicalName v, string name); // TODO: Can I deprecate this?
```

Comment:
Stray TODO.

I'm not sure what we're using it for, but type names don't have data flow nodes so there isn't an obvious replacement for it.

Reply:
The only use that does not look to be easily replaceable is `InterModuleTypeInference::AnalyzedVariableExport`.